frappe/utils/data.py

```diff
@@ -712,18 +880,33 @@ def validate_duration_format(duration):
 		)
 
 
-def get_weekdays():
+def get_weekdays() -> list[str]:
+	"""Return a list of weekday names.
+
+	Return value: ['Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday', 'Sunday']
```

Comment:
```suggestion
```

Seems unnecessary

Reply:
Since it doesn't take first weekday into consideration, it is better if we show the value in docs. Added specially for generated docs.

Reply:
Okay